ext/nnstreamer/tensor_filter/meson.build

```diff
@@ -204,7 +204,7 @@ if have_python2
     install_dir: nnstreamer_libdir
   )
 
-  shared_library('nnstreamer_python2',
+  nnstreamer_python2_lib = shared_library('nnstreamer_python2',
```

Comment:
Question: why do you add this?

Reply:
@jaeyun-jung These are WiP garbages and no more needed. Will be removed. Thanks.